components/Card/Card.module.scss

```diff
@@ -0,0 +1,72 @@
+.link {
+  cursor: pointer;
+  text-decoration: none;
+}
+
+.card {
+  box-shadow: 0 4px 20px 0 var(--shadow);
```

Comment:
px not rem?

Reply:
Good spot, I just took the values from Figma directly 🙂